src/Spectre.libGaClassifier/IndividualFeasibilityConditionsFactory.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+* IndividualFeasibilityConditionsFactory.cpp
+* Creates individual feasibility condition chain.
+*
+Copyright 2018 Spectre Team
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "IndividualFeasibilityConditionsFactory.h"
+#include "InconsistentMinimalAndMaximalFillupException.h"
+#include "Spectre.libGenetic/LengthCondition.h"
+#include "Spectre.libGenetic/MinimalFillupCondition.h"
+#include "Spectre.libGenetic/MaximalFillupCondition.h"
+#include "Spectre.libGenetic/MinimalPercentageFillupCondition.h"
+#include "Spectre.libGenetic/MaximalPercentageFillupCondition.h"
+#include "AllLabelTypesIncludedCondition.h"
+
+namespace spectre::supervised
+{
+IndividualFeasibilityConditionsFactory::IndividualFeasibilityConditionsFactory(size_t length, size_t minimalFillup, size_t maximalFillup, double minimalPercentageFillup,
+                                                                               double maximalPercentageFillup, bool allLabelTypes, gsl::span<const supervised::Label> labels):
```

Comment:
We *always* demand all label types for this kind of problems. It should not be a parameter, but rather private constant.

Reply:
ok, done